File: src/Vincent OS/Paramètres.vb
Public Class Paramètres

    Private Sub Options_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        If Bureau.Visible = True Then
            PictureBox1.Visible = True
            Label7.Text = "Utilisateur : Superutilisateur"
        ElseIf Bureau2.Visible = True Then
            PictureBox2.Visible = True
            Label7.Text = "Utilisateur : Invité"
        End If
    End Sub

    Private Sub Button3_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button3.Click
        About.Show()
    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs)
        Custom.Show()
    End Sub

    Private Sub Button4_Click(ByVal sender As System.Object, ByVal e As System.EventArgs)
        Terminal.Show()
    End Sub

    Private Sub Button4_Click_1(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button4.Click
        TaskVOS.Show()
    End Sub

    Private Sub Button1_Click_1(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Custom.Show()
    End Sub

    Private Sub Button5_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button5.Click
        Utilisateur.Show()
    End Sub
End Class

File: src/Vincent OS/Paramètres.cs
using System;

namespace Vincent_OS
{
    public partial class Paramètres
    {
        public Paramètres()
        {
            InitializeComponent();
        }

        private void Options_Load(object sender, EventArgs e)
        {
            Bureau bureau = new Bureau();
            Bureau2 bureau2 = new Bureau2();
            if (bureau.Visible == true)
            {
                PictureBox1.Visible = true;
                Label7.Text = "Utilisateur : Superutilisateur";
            }
            else if (bureau2.Visible == true)
            {
                PictureBox2.Visible = true;
                Label7.Text = "Utilisateur : Invité";
            }
        }

        private void Button3_Click(object sender, EventArgs e)
        {
            About about = new About();
            about.Show();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            Custom custom = new Custom();
            custom.Show();
        }

        private void Button4_Click(object sender, EventArgs e)
        {
            Terminal terminal = new Terminal();
            terminal.Show();
        }

        private void Button4_Click_1(object sender, EventArgs e)
        {
            TaskVOS taskVOS = new TaskVOS();
            taskVOS.Show();
        }

        private void Button1_Click_1(object sender, EventArgs e)
        {
            Custom custom = new Custom();
            custom.Show();
        }

        private void Button5_Click(object sender, EventArgs e)
        {
            Utilisateur utilisateur = new Utilisateur();
            utilisateur.Show();
        }
    }
}